### DISCUSSION TITLE
Black Screen when I Change

### DISCUSSION THREAD

**Discussion by jarpingdev** [MonitorControl/MonitorControl#760] [General]
I have a problem I have installed version 4.0 and my monitor detects a 29" LG ultrawide but when I change the volume or brightness the screen goes black, it is a bug?

**Comment by waydabber** [comment]
Does this happen when you uncheck "Use hardware DDC control" under Displays and then try to change brightness? If so then probably your display or display cabling or some other component does not tolerate DDC commands well.

Also, please describe your setup in more detail.

**Reply by jarpingdev** [reply]
I have seen that The problem is when I use hardware DDC control but if I uncheked hardware ddc control I lost slider volume control

**Reply by waydabber** [reply]
Yes, volume can be controlled via DDC only. This seems to be an issue with your setup it seems. What mac model, display, connection method etc model are you using exactly?

**Comment by jarpingdev** [comment]
My external monitor is LG 29WL50S-B and my macbook pro 13 2016 toutchbar

**Reply by waydabber** [reply]
That should generally work. I advise you to change the name of the discussion to reflect the topic so other LG 29WL50S-B users might chime in and let us know if DDC works for them.

You can also try other apps like [Lunar](https://lunar.fyi) (@alin23) or [ddcctl](https://github.com/kfix/ddcctl) to see if they can control the displays via DDC.

**Reply by jarpingdev** [reply]
Lunar not working with DDC

**Comment by joergRossdeutscher** [comment]
I have the same issue - sometimes. I still try to find out why and when the issue starts, but I am suspecting it has to do I close my MacBook before video meetings.

I have one of these MacBooks whose grafic card become very hot, and I can't have video chats with two screens. I bought and external webcam with microphone and an external monitor. Before a meeting I close the Macbook. Often when I try to change the volume in the meeting, the described behaviour appears.

I am a new user to Monitor Control and don't know if older versions had that already. Also, sorry, I am away from my equipment and cannot post the specs now. Let me know if this helps. And, not to forget: Thanks alot for this great tool!

**Comment by waydabber** [comment]
I don't know if this is the same issue as this but an other 29" LG user seems to have a similar problem:

https://github.com/MonitorControl/MonitorControl/discussions/773#discussioncomment-1606294
